runbotics/runbotics-orchestrator-ui/src/src-app/Actions/jira-server.actions.ts

```diff
@@ -1,11 +1,10 @@
 import { JiraServerAction, ActionRegex } from 'runbotics-common';
 
+import { ActionCredentialType } from '#src-app/credentials/actionCredentialType.enum';
```

Comment:
I've merged PR with ActionCredentialType moved to runbotics-common (cause it's being used both on scheduler and ui). Plase import it from there and remove definition from #src-app